### PR TITLE
chore: settings categories regrouped

### PR DESCRIPTION
**Pull request by Arvi3d**
### Description

This is a continuation of a work done in this PR earlier: https://github.com/snyk/vscode-extension/pull/496

The aim is to make an order of categories more logical according to usage pattern.
This is just an iteration. we will continue improving it.

There are few duplicating and possible unneeded settings left

### Checklist

- [ ] Tests added and all succeed
- [ ] Linted
- [ ] CHANGELOG.md updated
- [ ] README.md updated, if user-facing

### Screenshots / GIFs

_Visuals that may help the reviewer. Please add screenshots for any UI change. GIFs are most welcome!_

### CODE REVIEW

**Review by cat2608 — APPROVED** [review]
I think it makes sense! Thank you !! 🚀 

image

File: package.json [hunk 1]
```diff
@@ -297,6 +251,57 @@
             "markdownDescription": "Specifies whether to see only net new issues or known issues. Only applies to Code Security and Code Quality.\n\nNote: this is an experimental feature. Please reach out to [support.snyk.io](https://support.snyk.io) for more details."
           }
         }
+      },
+      {
+        "id": "onboarding",
```
Comment (bastiandoetsch) [inline package.json:256]: I don't like `Onboarding`. I'd chuck the welcome notification setting and just go for a category `Folders` and put Trust, Folderconfig and all paths there. @neilm-snyk , WDYT?

File: package.json [hunk 2]
```diff
@@ -297,6 +251,57 @@
...
+          "snyk.trustedFolders": {
+            "type": "array",
+            "default": [],
+            "description": "Folders to trust for Snyk scans."
+          }
+        }
+      },
+      {
+        "id": "cli",
+        "title": "Snyk CLI",
```
Comment (bastiandoetsch) [inline package.json:276]: LS is not CLI - I'd rather do it like above.

**Review by bastiandoetsch — CHANGES_REQUESTED** [review]
Let's discuss this before merging it - this is the first time I see this iteration on the configuration and I think we should discuss it :)

File: package.json (hunk 1 above)
Comment (Arvi3d) [reply package.json]: renamed to initialization

File: package.json (hunk 2 above)
Comment (Arvi3d) [reply package.json]: renamed to "CLI & Language server"

**Review by bastiandoetsch — APPROVED** [review]
works for me :)

**Review by cat2608 — APPROVED** [review]

**Pull request merged**